website/docs/callingconvention.md

```diff
@@ -1,17 +1,25 @@
+---
+title: callingconvention
+description: Sets whether or not the compiler should build STL modules.
```

Comment:
I'm against adding `title` and `description` to front-matters when they are not necessary. If not set, the title of the page is its filename and it's fine for 99% of our pages. Same goes for `description`. Docusaurus takes first paragraph as a description so there is no need to repeat it in the front-matter. It's bug-prone and this PR proves it. This description is copied from another page.  :)